content/en/docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -10,23 +10,45 @@ feature:
 
 {{% capture overview %}}
 
-When you specify a [Pod](/docs/concepts/workloads/pods/pod/), you can optionally specify how
-much CPU and memory (RAM) each Container needs. When Containers have resource
-requests specified, the scheduler can make better decisions about which nodes to
-place Pods on. And when Containers have their limits specified, contention for
-resources on a node can be handled in a specified manner. For more details about
-the difference between requests and limits, see
-[Resource QoS](https://git.k8s.io/community/contributors/design-proposals/node/resource-qos.md).
+When you specify a {{< glossary_tooltip term_id="pod" >}}, you can optionally specify how
+much resources each {{< glossary_tooltip text="Container" term_id="container" >}} needs.
+The most common resources you specify are CPU and memory (RAM), and there are others you might
```

Comment:
```suggestion
The most common resources you need to specify are CPU and memory (RAM), but there are others you might need to specify as well.
```

Reply:
This one I'm also not sure about; people never _need_ to specify requests and limits, but if they don't their cluster might not work how they'd like.

Reply:
Then maybe "the most common resources to specify are..."?